docs/userguide.rst

```diff
@@ -460,6 +460,9 @@ follows:
 
 We are done! Our outputs plot should be located in the ``results`` directory.
 
+Note that you can inspect your analysis workspace by using `Jupyter Nobebooks interactive sessions
```

Comment:
- by launching... or by opening, to make reference to the "open" command 
- Jupyter notebook interactive sessions